VueJS/index.js

```diff
@@ -3,11 +3,37 @@ import ListItem from './components/ListItem.js'
 
 new Vue({
   el: '#app',
+  components: {
+    List,
+    ListItem
+  },
   data: {
-    todos: [
+    lastId: 3,
+    todos:  [
       { id: '1', message: 'Foo', },
       { id: '2', message: 'Bar', },
       { id: '3', message: 'Baz', }
     ]
-  }
+  },
+  methods: {
+    newTodo: function() {      
+      this.lastId++
+      this.todos.push({id : this.lastId, message: ""})
+      // console.log("New Todo: [" + this.lastId + "]")     
+    },  
+    updateTodo: function(passedTodo) {
+        // console.log("(Großeltern) \nIch erhielt die todo [" + passedTodo.id + " | " + passedTodo.message + "] via $emit ")
+        // console.log("(Großeltern) \nFinde den entsprechenden index passend zur ID(" + passedTodo.id +") in todos[]")
+        let index = this.todos.findIndex( todo => todo.id == passedTodo.id)
```

Comment:
```suggestion
        let index = this.todos.findIndex( todo => todo.id === passedTodo.id)
```
Always use triple equation for comparisons like this: https://codeburst.io/javascript-double-equals-vs-triple-equals-61d4ce5a121a